llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -9445,6 +9445,42 @@ bool RISCVTargetLowering::isFMAFasterThanFMulAndFAdd(const MachineFunction &MF,
   return false;
 }
 
+bool RISCVTargetLowering::isJumpTableRelative() const {
+  if (!RISCVABI::isCheriPureCapABI(Subtarget.getTargetABI()))
+    return TargetLowering::isJumpTableRelative();
+
+  return true;
```

Comment:
Inverting this makes it easier to parse IMO.
```suggestion
  if (RISCVABI::isCheriPureCapABI(Subtarget.getTargetABI()))
    return true;

  return TargetLowering::isJumpTableRelative();
```

Reply:
This way it's consistent with getPICJumpTableRelocBase(Expr). Having this one inverted would make it more confusing when considering the file as a whole.